opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jdbc/JdbcTracingTest.java

```diff
@@ -0,0 +1,105 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@SpringBootTest(classes = {MockTracingConfiguration.class})
+@RunWith(SpringJUnit4ClassRunner.class)
+public class JdbcTracingTest {
+
+    @Autowired
+    MockTracer tracer;
+
+    @Autowired
+    DataSource dataSource;
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    @Before
+    public void before() {
+        tracer.reset();
+    }
+
+    /**
+     * Make sure we get a data source that returns a tracing connection. Without our Aspect, this would return a regular
+     * pooled connection for an embedded database (as we don't configure a database anywhere).
+     */
+    @Test
+    public void dataSourceIsInstrumented() throws SQLException {
+        assertTrue(dataSource.getConnection() instanceof TracingConnection);
+    }
+
+    /**
+     * Make sure we get one span once we execute a database call.
+     */
+    @Test
+    public void spanIsCreatedForPreparedStatement() throws SQLException {
+        PreparedStatement pstmt = dataSource.getConnection().prepareStatement("select 1");
+        assertTrue(pstmt.execute());
+        assertEquals(1, tracer.finishedSpans().size());
+    }
+
+    /**
+     * Make sure that a span is created when an active span exists joins the active
+     * @throws SQLException
+     */
+    @Test
+    public void spanJoinsActiveSpan() throws SQLException {
+        try (ActiveSpan ignored = GlobalTracer.get().buildSpan("parent").startActive()) {
+            assertTrue(dataSource.getConnection().prepareStatement("select 1").execute());
+            assertEquals(1, tracer.finishedSpans().size());
+        }
+
+        assertEquals(2, tracer.finishedSpans().size());
+        Optional<MockSpan> jdbcSpan = tracer
+                .finishedSpans()
+                .stream()
+                .filter((s) -> "java-jdbc".equals(s.tags().get(Tags.COMPONENT.getKey())))
+                .findFirst();
+
+        Optional<MockSpan> parentSpan = tracer
+                .finishedSpans()
+                .stream()
+                .filter((s) -> "parent".equals(s.operationName()))
+                .findFirst();
+
+        assertTrue(jdbcSpan.isPresent());
+        assertTrue(parentSpan.isPresent());
+
+        assertEquals(jdbcSpan.get().context().traceId(), parentSpan.get().context().traceId());
+        assertEquals(jdbcSpan.get().parentId(), parentSpan.get().context().spanId());
+    }
+
+    /**
+     * Make sure that a span is created when executing statements via a JDBC template
+     */
+    @Test
+    public void spanIsCreatedWhenUsingJdbcTemplate() {
```

Comment:
the PR looks awesome. But could you please also verify here is wiring with parent span works here?